src/twisted/python/test/test_release.py

```diff
@@ -1400,7 +1412,7 @@ def test_remove(self):
         """
         reposDir = self.makeRepository(self.tmpDir)
         testFile = reposDir.child('some-file')
-        testFile.setContent("something")
```

Comment:
Absent further context, I would likely be convinced by the private interface argument here.  However, I have observed a number of regressions introduced by Python 3 porting work in recent Twisted releases (a much higher rate than I've ever observed from other kinds of development) and to me this signals a need to be more careful in further porting efforts.  It's _less_ work to address this properly now than to land something half-complete and then, should the need arise, deal with bug fixes and potentially issuing bug-fix releases.

It also seems to me that at this point the way to port `str`-using code should be quite well understood and there should be no real blockers to porting this code according to those methods.